PSOperationsTests/PSOperationsTests.swift

```diff
@@ -973,4 +1006,40 @@ class PSOperationsTests: XCTestCase {
         XCTAssertEqual(1, opQ.operationCount)
         XCTAssertTrue(op.waitCalled)
     }
+    
+    func testOperationsNotExecuting() {
+        
+        var opCount = 0
```

Comment:
@timshadel Not sure I fully understand. The nature of this test is that if we reach the timeout and the `opCount` doesn't equal the `requiredToPassCount` it means the queue is frozen. This test only adds one item to the queue at a time as well. So once we freeze we have 1 op that is frozen in the queue and no other ops in the queue.